requirements.txt

```diff
@@ -1,3 +1,8 @@
+asgiref==3.5.2
 Django==3.2
+djangorestframework==3.13.1
+djangorestframework-simplejwt==5.2.0
```

Comment:
Remove asgiref, PyJWT, pytz, sqlparse only djangorestframework-simplejwt should be in changes

Reply:
ok